test/e2e/elasticsearch_test.go

```diff
@@ -80,22 +81,38 @@ func (suite *ElasticSearchTestSuite) TestSparkDependenciesES() {
 }
 
 func (suite *ElasticSearchTestSuite) TestSimpleProd() {
+	jaegerInstanceName := "simple-prod"
+	runSimpleProdTest(jaegerInstanceName, false)
+}
+
+func (suite *ElasticSearchTestSuite) TestSimpleProdWithOTEL() {
```

Comment:
Do we need to have separate tests for OTEL?  Isn't it enough to have configurable env var for OTEL and then run the subset of tests with changed config?

Reply:
How would that work in CI?  I was assuming we'd want to run both sets of tests for the time being.

Reply:
The CI job would set an env var to use OTEL distributions. The same test would run with a different config.